src/main/java/sirius/web/security/oauth/OAuth.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.security.oauth;
+
+/**
+ * Defines some constants used for OAuth2 authentication.
+ */
+public class OAuth {
+
+    private OAuth() {
+        // No instances allowed
+    }
+
+    /**
+     * The token type bearer. It gets prepended in the authentication header and returned with the valid token.
+     *
+     * @see #OAUTH_TOKEN_TYPE
+     */
+    public static final String TOKEN_TYPE_BEARER = "bearer";
+
+    /**
+     * Determines the access token used to access the API.
+     */
+    public static final String OAUTH_ACCESS_TOKEN = "access_token";
+
+    /**
+     * Determines the client id that determines an OAuth client.
+     */
+    public static final String OAUTH_CLIENT_ID = "client_id";
```

Comment:
**Optional:** In my opinion prefixing constants in the `OAuth` class with `OAUTH` is a bit redundant :)